### DISCUSSION TITLE
GitHub pages problem<title>

### DISCUSSION THREAD

**Discussion by Janardhan543** [community/community#61707] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

In my GitHub account,the GitHub pages are showing the problem as "Deceptive site ahead" ,can you please help me how to resolve this problem

**Comment by nirav-gajera** [comment]
The "Deceptive site ahead" warning is displayed by Google Chrome when it detects that a website contains social engineering content. This could include phishing pages, malware, or other types of malicious content.

There are a few things you can do to try to resolve this problem:

1. **Check your website for malicious content.** Use a website scanner to scan your website for malware or other malicious content. If you find any malicious content, remove it immediately.
2. **Make sure your website is up to date.** Outdated websites are more likely to be hacked, which could lead to the "Deceptive site ahead" warning being displayed. Make sure your website is up to date with the latest security patches.
3. **Contact Google Safe Browsing.** If you've checked your website for malicious content and it's still showing the "Deceptive site ahead" warning, you can contact Google Safe Browsing to request that your website be removed from the block list.

Here are the steps on how to contact Google Safe Browsing:

1. Go to the Google Safe Browsing website: https://safebrowsing.google.com/.
2. Click on the "Report a harmful site" button.
3. Enter the URL of your website in the "Site address" field.
4. Select the "Deceptive site" option from the "Type of issue" drop-down menu.
5. Click on the "Submit" button.
